input-output-hk/daedalus
# [DDW-277] Fix Download Manager directory creation error

This PR fix the current issue with the Download Manager, where it will not create the proper `Downloads` directory, which will prevent Daedalus to load.

## Screenshots

<img width="420" alt="Screen Shot 2020-06-15 at 09 48 44" src="https://user-images.githubusercontent.com/1504716/84662925-d5c50900-aef2-11ea-9ec2-5b370c91a383.png">


---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1592228040046300)
1. Delete the `stateDirectoryPath` (it can be found in the Diagnostics screen);
2. Run Daedalus;
3. The error message in the image above should not appear and Daedalus should run normally.

---

## Review Checklist

### Basics

- [ ] PR has a good description that summarizes all changes
- [ ] PR has default-sized Daedalus window screenshots or animated GIFs of important UI changes:
  - [ ] In English
  - [ ] In Japanese

### Code Quality

### Testing

### After Review
